### ISSUE TITLE
Typescript compile error 2. Empty error

### ISSUE THREAD

**Issue by xReveres** [denoland/deno#5769]
Simple steps to reproduce:
Run this script:
```typescript
function test(a: number, b: number): number{
    return a + b;
}
test(, 5);
```
Then, after run, we receive empty error.
```
error: 
```

**Comment by caspervonb** [comment]
Missing some details here I think? errors as expected for me.

```shell
Compile file:///Users/caspervonb/Projects/5769.ts
error: TS2554 [ERROR]: Expected 2 arguments, but got 1.
test(, 5);
~~~~~~~~~
    at file:///Users/caspervonb/Projects/5769.ts:4:1

    An argument for 'b' was not provided.
    function test(a: number, b: number): number{
                             ~~~~~~~~~
        at file:///Users/caspervonb/Projects/5769.ts:1:26

TS1135 [ERROR]: Argument expression expected.
test(, 5);
     ^
    at file:///Users/caspervonb/Projects/5769.ts:4:6

Found 2 errors.
```

**Comment by kitsonk** [comment]
Duplicate of #5739 

Fixed in #5752, please update to Deno 1.0.2.

**Comment by xReveres** [comment]
Yes, fine sorry. Because yesterday i saw commit for v1.0.2, but deno upgrade said i'm using the newest version.
Now:
```
error: Expected unicode escape at file:///home/ubuntu/workspace/deno/test.ts:4:0
```
